src/lib/devConsole.js

```diff
@@ -1000,9 +952,9 @@ const getMeshDeployments = async (organizationCode, projectId, workspaceId, mesh
 		logger.error(`Error fetching deployments for mesh: ${meshId}`);
 
 		return {
-			status: 'ERROR',
```

Comment:
The `error` status has special meaning in the deployments table. It signifies that an error occurred during provisioning. 

However, an error here signifies that we were unable to fetch the deployment status altogether. Thus, we don't want to default to that status here. Changed to `null` instead